.github/workflows/render-deployment.yml

```diff
@@ -0,0 +1,27 @@
+name: Compstore UI Continuous Delivery
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  test_pull_request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "18"
+
+      # https://github.com/marketplace/actions/render-deploy-action
+
+      - name: Deploy to production
+        uses: johnbeynon/render-deploy-action@v0.0.8
+        with:
+          service-id: ${{ secrets.MY_RENDER_SERVICE_ID }}
+          ui-key: ${{ secrets.MY_RENDER_UI_KEY }}
```

Comment:
Do procesu deploy na render nic nie dodawałem bo kod na master już jest sformatowany, a nawet jak by nie był to i tak jak będzie robiony z niego build to nie będzie "pretty".

Reply:
Spoko, i tak przycisk Merge będzie nieaktywny dopóki to sprawdzenie z workflow `test_pull_request` nie wykona się poprawnie